src/wagtail_localize_smartling/utils.py

```diff
@@ -21,19 +21,40 @@ def format_smartling_locale_id(locale_id: str) -> str:
     Format a locale ID for the Smartling API. Wagtail/Django use lower case
     (e.g. "en-us") whereas Smartling uses lower case for the language code and
     upper case for the region, if any (e.g. "en-US").
+
+    This behaviour is not applied if the REFORMAT_LANGUAGE_CODES setting is False
+
+    Also, this applies any mapping defined by the LOCALE_MAPPING_CALLBACK or
+    LOCALE_TO_SMARTLING_LOCALE settings.
     """
+    # Apply any mapping defined in settings
+    locale_id = smartling_settings.LOCALE_TO_SMARTLING_LOCALE.get(locale_id, locale_id)
+
+    if smartling_settings.REFORMAT_LANGUAGE_CODES is False:
+        return locale_id
```

Comment:
Issue: I'm not sure we ever want to skip the re-casing of codes that we send to Smartling; the casing there seems to be consistently `aa-BB`. I am sure that if we _do_ want to able to do that, we shouldn't be using the same setting/flag that we use to control re-casing of Django language codes.

Reply:
Yeah I can see where you're coming from. Happy for us to back that one out - it's the Django-side that needs a bit more flexibility

Reply:
This should teach me to do drive-by reviews. Given Smartling expects it a specific format, it is the reverse that we should (optionally) normalize 🤦🏼  I shall go in the corner in shame